src/manager/hydraPaper.ts

```diff
@@ -1,23 +1,33 @@
+import Gio from 'gi://Gio';
+
 import * as Utils from '../utils.js';
+import {Settings} from './../settings.js';
 
 import {ExternalWallpaperManager} from './externalWallpaperManager.js';
 
+// https://gjs.guide/guides/gjs/asynchronous-programming.html#promisify-helper
+Gio._promisify(Gio.Subprocess.prototype, 'communicate_utf8_async', 'communicate_utf8_finish');
+
```

Comment:
It builds for me without these changes. And as you mentioned the girs module has a declaration for the promisified version as well.

Did you get a runtime error?

Reply:
Yes, a runtime error complaining about too few parameters. (something similar to: expected 3, got 2)

Reply:
Alright, we have some promisify stuff anyway in the code somewhere so we can just keep it IMO.